iroha-cli/interactive/impl/interactive_query_cli.cpp

```diff
@@ -58,17 +58,17 @@ namespace iroha_cli {
       const auto role_id = "Requested role name";
       const auto tx_hashes = "Requested tx hashes";
 
-      query_params_descriptions_ = {
-          {GET_ACC, {acc_id}},
-          {GET_ACC_AST, {acc_id, ast_id}},
-          {GET_ACC_AST_TX, {acc_id, ast_id}},
-          {GET_ACC_TX, {acc_id}},
-          {GET_TX, {tx_hashes}},
-          {GET_ACC_SIGN, {acc_id}},
-          {GET_ROLES, {}},
-          {GET_AST_INFO, {ast_id}},
-          {GET_ROLE_PERM, {role_id}}
-          // query_params_descriptions_
+      query_params_map_ = {
+          {GET_ACC, makeParamsDescription({acc_id})},
+          {GET_ACC_AST, makeParamsDescription({acc_id, ast_id})},
+          {GET_ACC_AST_TX, makeParamsDescription({acc_id, ast_id})},
+          {GET_ACC_TX, makeParamsDescription({acc_id})},
+          {GET_TX, makeParamsDescription({tx_hashes})},
+          {GET_ACC_SIGN, makeParamsDescription({acc_id})},
+          {GET_ROLES, makeParamsDescription({})},
+          {GET_AST_INFO, makeParamsDescription({ast_id})},
+          {GET_ROLE_PERM, makeParamsDescription({role_id})}
+          // query_params_map_
```

Comment:
i think it can be removed

Reply:
Could you please clarify, not sure what you meant here.

Reply:
I mean the line with a comment. But you don't really have to remove it since the PR is about different things

Reply:
No, this comment is needed for better formatting